ts/parts/SvgRenderer.ts

```diff
@@ -976,7 +978,9 @@ extend((
             if (complete) {
                 animOptions.complete = complete;
             }
-            animate(this, params, animOptions);
+            H.syncTimeout((): void => {
```

Comment:
Load `syncTimeout` from `Utils` as shown in https://github.com/highcharts/highcharts/commit/4aa3cff7e9277f5648f209c8fd671e3a9de1b026